README.md

```diff
@@ -25,33 +29,14 @@ This repo is the package manager client for the Dynamo Package Manager.  Package
 
 - From Visual Studio IDE:
 
-    Build `The PackageManagerClient\src\GregClient.sln` solution
+    Build the `PackageManagerClient\src\GregClient.sln` solution
 
 - From command line:
 
     ```bat
     nuget restore src\GregClient.sln
     msbuild src\GregClient.sln
     ```
-
-#### Nuget
-The Package Manager Client is available from NuGet [here](https://www.nuget.org/packages/Greg/). 
-
-#### Manual Build of the NugetPackage
-- Make sure you have Nuget version 4.9.0 or later.
->**Important Note:** Please avoid to publish a locally built package
-
-##### Steps:
-
-1. Build the solution
-2. On the repository directory, execute the following PowerShell script with the following parameters
-   ```bat
-   pwsh -ExecutionPolicy ByPass -File ".\nuspec\ManualBuildPackage.ps1 [REPOSITORY_PATH] [NUGET_EXE_PATH] [OUTPUT_PATH]"
```

Comment:
Just want to double check, such file no longer exist right?

Reply:
Yes. Deleted here https://github.com/DynamoDS/PackageManagerClient/pull/61/files#diff-cc61b237cdc9569dcc6ee861344c0a65a9bd379a00510e71589163502fa4e1d1